src/Features/Core/Portable/QuickInfo/QuickInfoItem.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.QuickInfo
+{
+    public sealed class QuickInfoItem
+    {
+        /// <summary>
+        /// The span of the document that the item is based on.
+        /// </summary>
+        public TextSpan Span { get; }
+
+        /// <summary>
+        /// Descriptive tags from the <see cref="Completion.CompletionTags"/> type.
```

Comment:
Should this be WellKnownTags? It looks like CompletionTags is just a forward to that now...I feel like we had completiontags first and then realized it should have been generalized?

Reply:
Moved to `WellKnownTags`. I'm *really* not thrilled that `WellKnownTags` is a type in its own namespace, causing users to have to add a using just to interact with completion and quick info.

New public APIs need to go through a public API review for good reason.